torch/csrc/jit/codegen/onednn/graph_helper.cpp

```diff
@@ -585,27 +584,31 @@ LlgaNodeWrapper::LlgaNodeWrapper(const Node* node)
 }
 
 void LlgaNodeWrapper::setOpaqueLayout(size_t offset) {
-  TORCH_CHECK(offset < n->outputs().size(), "Invalid output offset ", offset);
+  const auto num_output = n->is(attr::output_layouts).size();
+  TORCH_CHECK(
+      offset < num_output,
+      "Out of range. (Invalid index ",
+      offset,
+      " for attr::output_layouts with size ",
+      num_output,
+      ")");
   auto& layouts =
       const_cast<std::vector<int64_t>&>(n->is(attr::output_layouts)); // NOLINT
   layouts.at(offset) = 1;
```

Comment:
`1` here means opaque layout? If so, should we use some constant or macro instead of a magic number? Same comment for the usage of `0` in the code.